crates/nu-command/Cargo.toml

```diff
@@ -92,10 +93,14 @@ toml = "0.7"
 unicode-segmentation = "1.10"
 ureq = { version = "2.7", default-features = false, features = ["charset", "gzip", "json", "native-tls"] }
 url = "2.2"
+uu_cp = "0.0.19"
+# uu_cp = { path = "../../../coreutils/src/uu/cp", version = "0.0.19" }
+uu_mv = "0.0.19"
+# uucore = { path = "../../../coreutils/src/uucore", version = "0.0.19", features = ["fsext"] }
+uucore = { version = "0.0.19", features = ["fsext"] }
```

Comment:
If `fsext` is necessary, that would be a bug in `uutils`. Does it still compile if you remove this?

Reply:
I'll test this and let you know. I didn't add it because it was required. I just copy-n-pasted from somewhere. It could've been from JT's exploration of adding `df` a few months back.